### PR TITLE
Fixes non-parallax example

### LINKED ISSUE #7375: Fix non-parallax example on 'Create a scrolling parallax effect' page
### Page URL

https://docs.flutter.dev/cookbook/effects/parallax-scrolling/

### Page source

https://github.com/flutter/website/tree/main/src/cookbook/effects/parallax-scrolling.md

### Describe the problem

The comparison between non-parallax & parallax effects is not perfect because the code offered for non-parallax:

```dart
  Widget _buildParallaxBackground(BuildContext context) {
    return Image.network(
      imageUrl,
      fit: BoxFit.cover,
    );
  }
```

will create an image that does not fill the whole parent stack widget, see:

image

### Expected fix

To improve this, the code should be:

```dart
  Widget _buildParallaxBackground(BuildContext context) {
    return Positioned.fill(
        child: Image.network(
      imageUrl,
      fit: BoxFit.fill,
    ));
  }
```

Thus we can get the desired screen:

image

### Additional context

More appropriate usage example of Stack, as well as its child widgets, can give newbies a better understanding of flutter, and also can eliminate the time every developer spends to solve this corner case problem.

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #7375

### CODE REVIEW

File: examples/cookbook/effects/parallax_scrolling/lib/excerpt2.dart [hunk 1]
```diff
@@ -35,10 +35,11 @@ class LocationListItem extends StatelessWidget {
   }
 
   Widget _buildParallaxBackground(BuildContext context) {
-    return Image.network(
+    return Positioned.fill(
+        child: Image.network(
       imageUrl,
       fit: BoxFit.cover,
-    );
+    ));
```
Comment (johnpryan) [inline examples/cookbook/effects/parallax_scrolling/lib/excerpt2.dart:42]: nit: add a trailing comma for the formatter:

```dart
return Positioned.fill(
  child: Image.network(
    // ...
  ),
);

**Review by Piinks — APPROVED** [review]
Flutter_LGTM

**Comment by atsansone** [comment]
@johnpryan : Back to you!

**Review by domesticmouse — APPROVED** [review]
Formatting LGTM

**Review by parlough — APPROVED** [review]

**Pull request merged**